spk/minio/src/wizard/install_uifile

```diff
@@ -0,0 +1,64 @@
+[
+	{
+		"step_title": "MinIO configuration",
+		"items": [
+		  {
+			"type": "textfield",
+			"subitems": [
+			  {
+				"key": "wizard_data_directory",
+				"desc": "Data directory location",
+				"defaultValue": "/volume1/minio",
+				"validator": {
+				  "allowBlank": false,
+				  "regex": {
+					"expr": "/^\\\/volume[0-9]{1,2}\\\/[^<>: */?\"]*/",
+					"errorText": "Path should begin with /volume?/ where ? is volume number (1-99)"
+				  }
+				}
+			  },
+			  {
+				"key": "wizard_access_key",
+				"desc": "MinIO access key",
+				"defaultValue": "minio-access-822791549661344",
```

Comment:
I propose to use the simple default value "minio" as used in some minio examples.
Your default value looks like containing a random number but it does not! It will be the same for all package installations.

Reply:
Wouldn't it be better to not set a default value at all from a security point of view. Then people are really forced to use their own keys and it is their choice to use strong or weak ones.

Reply:
My idea was also to make them empty by default to force the user to use his own password. I just added those values to make testing more convenient. Fixed in https://github.com/SynoCommunity/spksrc/pull/3689/commits/fb5b9e9813718d4052368d31117ea126ecc6b703